front_end/src/views/AboutView.vue

```diff
@@ -0,0 +1,5 @@
+<template>
```

Comment:
If this is just boiler plate from the generator and not going to be used it can be removed from the commit.